packages/components-js/tests/vrt/playwright/helpers/get-html-for-reference-id.ts

```diff
@@ -0,0 +1,6 @@
+import type { Page } from '@playwright/test';
+import { selectNode } from '../../../e2e/puppeteer/helpers';
+
+export const getHtmlForReferenceId = async (page: Page, selector: string) => {
```

Comment:
Strange name, also return type is missing.
Isn't this returning the value of the `for` attribute?